app/main.py

```diff
@@ -1 +1,7 @@
-# write your code here
+def copy_file(command: str) -> None:
```

Comment:
The function should verify that the command starts with 'cp' and contains exactly three elements separated by spaces. This is required to ensure the command is correctly formatted according to the task description and checklist.